### DISCUSSION TITLE
[Bookmarks] Import one/more KML files; Delete all selected; Rename "List Settings"

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#1615] [Ideas]
Hello,

I have the following suggestions for the Bookmarks section:
1. Instead of importing all KML files from a given folder, make it possible to only import selected files within
2. Delete all selected sets: Currently, the user must delete each set one at a time (Bonus: Add an "Undo" button…)
3. Rename "List Settings" as… "Rename"

Cheers,

![image](https://user-images.githubusercontent.com/1541861/143463651-f07179f3-f86c-45eb-8b20-3f889a42da8b.png)

**Comment by biodranik** [comment]
1. Any file can be clicked and open/imported in OM. Why do you need to use Import Bookmarks button for single files? What is your use-case?
2. If you import only one set (assuming (1) above is fixed), do you still need to mass-delete sets? What is your use-case?
3. We probably add there more sharing-related options.
